README.md

```diff
@@ -1,4 +1,4 @@
-# Awesome Flatpak
+# Awesome Flatpak  [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
```

Comment:
This repo isn't on awesome.re though. If they add us, then we can have the link.

Reply:
okay, i add this list in [awesome.re](https://awesome.re/)